tests/argparse/test_argparse.py

```diff
@@ -307,3 +304,12 @@ def test_parse_args(self) -> None:
         # Check that the error is handled properly
         with raises(SystemExit):
             parser.parse_args(["--src_host", "host", "--src_port", "42", "--src_user", "username"])
+
+    @pytest.mark.dependency(depends=["add_server_arguments"])
+    def test_parse_args_not_server_host(self) -> None:
+        """Tests `ArgumentParser.parse_args()` method when a non-database server host is added as argument."""
```

Comment:
Just to understand more clearly that this test matches based on prefix, do we want to add another parameter to test what happens then the arg is like 'file_hostdb' ?
```suggestion
        """Tests `ArgumentParser.parse_args()` method when a non-database server host is added as argument.
        This test checks that arguments with a `host` suffix in their names (e.g., `file_host`) are parsed correctly. 

        """
```